boards.stm32.mk

```diff
@@ -116,6 +116,7 @@ $(eval $(call stm32f4board,nucleo-f410rb,GPIOA,GPIO5))
 $(eval $(call stm32f4board,nucleo-f446re,GPIOA,GPIO5))
 $(eval $(call stm32f4board,olimex-p405,GPIOC,GPIO12))
 $(eval $(call stm32f4board,olimex-e407,GPIOC,GPIO13))
+$(eval $(call stm32f4board,black-stm32f4xx,GPIOA,GPIO6))
```

Comment:
can we call it the black stm32f407ve board at least?  https://forum.micropython.org/viewtopic.php?t=3086 mentions that the 407ve and the 407ze boards have the led on different pins, (PA6/7 vs PF9/10 for led1/2 respectively)

Reply:
https://jeelabs.org/2018/retrocomputing-on-f407/ shows a 407vg board with the led on PE0 too, so "f4xx" is not really a good enough id for your board I'm afraid :)

Reply:
You are right. My board is labeled as `STM32_F4VE v2.0` build `1509`.
Should I add version number in addition to the chip name?

Reply:
version seems useful, can probably skip the build 1509...